sandpack-react/src/contexts/sandpackContext.test.tsx

```diff
@@ -54,6 +54,38 @@ describe(SandpackProvider, () => {
       ]);
     });
 
+    it("deletes the activeFile and set the following visibleFile as active", () => {
+      const root = create(
+        <SandpackProvider
+          template="react"
+          options={{ activeFile: "/App.js", visibleFiles: ["/styles.css"] }}
+        />
+      );
+
+      const instance = root.getInstance();
+
+      instance.runSandpack();
+      instance.deleteFile("/App.js");
+
+      expect(instance.state.activeFile).toBe("/styles.css");
+    });
+
+    it("deletes the activeFile and set the entry file if there no visibleFile left", () => {
```

Comment:
With your change does this test need to be modified as we don't expect just index.js